src/components/styles/assetsStyles.css

```diff
@@ -0,0 +1,104 @@
+body {
+  margin: 0;
+  padding: 0;
+  background-color: #4369b2;
+  font-family: 'Lato', sans-serif;
+}
+
+.header {
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  z-index: 0;
+  background-color: #4369b2;
+}
+
+.container {
+  margin: 28vh 0 0 0;
+  color: white;
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  width: 100vw;
+}
+
+.title {
+  margin: 12vh 0 5vh 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  /* height: 18vh; */
+  color: white;
+  text-transform: uppercase;
+  font-size: 3rem;
+}
+
+.subtitle {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: #0c2553bb;
+  color: white;
+  padding-left: 0.5rem;
+  font-weight: 700;
+  font-size: 0.9rem;
+  height: 4vh;
+}
+
+/* .asset:nth-child(1) {
+    margin: 0;
+    padding: 0;
+    grid-column: 1 / 3;
+    grid-row: 1 / 2;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-around;
+    align-items: center;
+    height: 25vh;
+  } */
+
+.asset {
+  height: 24vh;
+  margin: 0;
+  padding: 2vw;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+
+  /* border: solid 1px blue; */
+
+  /* padding: 1rem; */
+
+  /* margin: 0.5rem; */
```

Comment:
- [ ] Good job so far 👍🏽 . i strongly think it will nice if you dont leave any commented code in your codebase so that your codebase will be clean, readable and maintainable.

Reply:
Hi! Its done 🔨